### ISSUE TITLE
Tabbing between multiple IFrame instances

### ISSUE THREAD

**Issue by indeox** [sanity-io/sanity-plugin-iframe-pane#82]
Not sure if this is a bug, or if it was even ever intended to be used this way. I'm trying to set up multiple preview panes (for example to preview the document _and_ a separate tab for Social Cards).

However, the iframe doesn't seem to switch URL when flipping between them. However, it _does_ change if you tab back to the main form, and _then_ switch to a different tab.

The issue only seems to happen if you switch directly between IFrame tabs.

Here's a reduced example:
```js
S.document().views([
  // Main form view
  S.view.form().icon(EditIcon),

  // Document preview
  S.view
    .component(Iframe)
    .options({
      url: `https://placehold.co/600x400/9b59b6/white?text=Preview+A`,
    })
    .title("Preview A")
    .icon(EyeOpenIcon),

  // Social cards preview
  S.view
    .component(Iframe)
    .options({
      url: `https://placehold.co/600x400/2980b9/white?text=Preview+B`,
    })
    .title("Preview B")
    .icon(EyeOpenIcon),
]);

```

And here's  a video of what happens:

https://github.com/sanity-io/sanity-plugin-iframe-pane/assets/518943/789854b4-b2e3-4a72-9c92-10f0a5a96dfc

Is there any way to work around this? Thank you!

**Comment by ehuelsda** [comment]
I have the same issue. A fix would be great!

**Comment by stipsan** [comment]
Solved in `v3.1.0`:
```ts
S.document().views([
  // Main form view
  S.view.form().icon(EditIcon),

  // Document preview
  S.view
    .component(Iframe)
    .options({
      key: 'a',
      url: `https://placehold.co/600x400/9b59b6/white?text=Preview+A`,
    })
    .title("Preview A")
    .icon(EyeOpenIcon),

  // Social cards preview
  S.view
    .component(Iframe)
    .options({
      key: 'b',
      url: `https://placehold.co/600x400/2980b9/white?text=Preview+B`,
    })
    .title("Preview B")
    .icon(EyeOpenIcon),
]);
```

**Comment by indeox** [comment]
That's great news! Thanks so much. Legend 🎉
